src/components/mensJacket/MensJacket.js

```diff
@@ -16,11 +17,14 @@ const MensJacket = () => {
             <span>AMAYRA'S SHOP CHOICE</span>
             </div>
             <div className="mens-jacket-cont">
-            <ColorCard />
-            <ColorCard />
-            {/* <RatedCard imgOne="/images/card/shirt-1.jpeg" name={"ELEMENT BUFFALO PLAID FLANNEL SHIRT IN REGULAR FIT"}/> */}
-            <ColorCard />
-            {/* <PriceCard imgOne={"/images/card/shirt-2.jpeg"} name={"SELECTED HOME OVERCOAT"}/> */}
+            {products.slice(6,9).map((val,index)=>{
+                return(
+                    <div key={index}>
+                    <PriceCard pID={val._id} imgSrcTwo={val.imgTwo} imgSrcOne={val.imgOne} stock={val.stock} name={val.title} type={val.type} price={val.price}/>
```

Comment:
IMO, instead of sending destructured properties here, you can send the object itself and destructure it in PriceCard component that would look cleaner. Also, this way you can reduce repetitive code.